src/js/components/modals/CliInstallModal.js

```diff
@@ -18,9 +18,12 @@ class CliInstallModal extends React.Component {
   constructor() {
     super(...arguments);
 
-    this.state = {
-      selectedOS: browserInfo().os
-    };
+    let selectedOS = browserInfo().os;
```

Comment:
I know it's confusing, but this works. If you do `linux` lowercase it won't work.